app/api/attest/volume/route.ts

```diff
@@ -0,0 +1,68 @@
+import { type NextRequest, NextResponse } from "next/server";
+import { auth } from "@/lib/auth";
+import { signVolume } from '@/lib/signing/volume';
+
+import { getRedisInstance } from "@/lib/redis";
+import { Lock } from "@upstash/lock";
+import { ATTESTATION_DEADLINE } from "@/lib/config";
+import { getWalletAddress } from "@/lib/utils";
+
+import { ethers, Wallet, JsonRpcProvider } from 'ethers';
+import { getTotalVolume, getAttestedVolume } from "@/lib/volume";
+import { getProxy } from "@/lib/proxy";
+
+export const runtime = "edge";
+
+
+export async function POST(req: NextRequest) {
+  const session = await auth();
+  if (session === undefined || session === null) {
+    return NextResponse.json({ error: 'Not authenticated' }, { status: 400 })
+  }
+
+  const walletAddress = getWalletAddress(session)
+  if (!walletAddress) {
+    return NextResponse.json({ error: 'No wallet address found in session' }, { status: 400 })
+  }
+
+  const provider = new JsonRpcProvider(process.env.RPC_PROVIDER);
```

Comment:
You can use the importes JSON_RPC_ENDPOINT from lib/config, like this: https://github.com/avax-attestations/passport-frontend/blob/main/lib/diamond-hands.ts#L23